.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+/.github/stable/ @PennyLaneAI/Core
```

Comment:
It was chosen arbitrarily by me, happy to move it - I chose it for 2 reasons:
- it's maintained by github actions, so it felt right
- I don't want people outside of the team to really know about it, so this is a nice place to hide things

If you have a preference for a new place, lmk!

Reply:
After looking into it a bit more, the consensus seems to be that the `.github` directory is for files related to GitHub, so we could make the claim that this is fine since the files are only touched by GitHub Actions. We could also just make a root-level `requirements` or `build` directory, but it may be nice to keep things hidden as you said.